src/validation/custom.validation.ts

```diff
@@ -661,3 +679,8 @@ export const checkFieldIfRadioButtonSelectedAndFieldsEmpty = (isPrimaryField: bo
     }
   }
 };
+
+export const customValidations = {
```

Comment:
you shouldn't need to export it again as the function is already exported on line 540

Reply:
I needed to alias the module in order to use it in the spy function:
`jest.spyOn(customValidations, "isUnableToObtainAllTrustInfo");`
However, it's not mocking properly and as the functionality is being tested elsewhere, I have removed the test.